exporter/collector/metricsexporter.go

```diff
@@ -173,7 +175,35 @@ func attributesToLabels(
 	attrs pdata.AttributeMap,
 ) labels {
 	// TODO
-	return nil
+	ls := make(labels, attrs.Len())
+	attrs.Range(func(k string, v pdata.AttributeValue) bool {
+		ls[sanitizeKey(k)] = v.AsString()
+		return true
+	})
+	return ls
+}
+
+// Replaces non-alphanumeric characters to underscores. Note, this does not truncate label keys
+// longer than 100 characters or prepend "key" when the first character is "_" like OpenCensus
```

Comment:
Should we start a list somewhere of the changes between OC and otel exporters?

Reply:
Yup, let me add some markdown

Reply:
Added `exporter/collector/breaking-changes.md`